src/FeedsCleanup/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Web.Xdt" version="2.1.0" targetFramework="net452" />
+  <package id="NDesk.Options" version="0.2.1" targetFramework="net452" />
```

Comment:
Allow me hit the break a little bit. Let's not keep snow balling the features here. I file this issue to try the global command suggestion. https://github.com/aspnet/Coherence/issues/26